### PR TITLE
Add a section for Status downloads

### PR DESCRIPTION
**Pull request by jinhojang6**
[Need more reviews]

- Added the `Download Status for` section.
test: https://es-blog.status.im/lanzamiento-v1-10-dapp-favoritas-e-imagenes-de-perfil/

Screen Shot 2021-01-27 at 1 20 55 AM

- Added `Open positions` menu to `Get involved`

### CODE REVIEW

File: post.hbs [hunk 1]
```diff
@@ -81,6 +81,60 @@
...
+                                <div class="download-title">Mobile</div>
+                                <div class="download-description">Private messaging, secure crypto transactions, Web3 DApp browsing in your pocket.</div>
+                                <div class="download-buttons">
+                                    <a href="https://apps.apple.com/us/app/status-private-communication/id1178893006" id="official:ios" class="">
+                                        <img class="mobile-download" src="{{asset "img/app-store.svg"}}">
+                                    </a>
+                                    <a href="https://play.google.com/store/apps/details?id=im.status.ethereum" id="official:android" class="download-margin-left">
+                                        <img class="mobile-download" src="{{asset "img/google-play.svg"}}">
+                                    </a>
+                                    <a href="https://status-im-files.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-v1.10.1.apk" id="official:apk" class="download-margin-left">
```
Comment (jakubgs) [inline post.hbs:100]: Do we REALLY want YET ANOTHER place to keep updating whenever releases change?

Comment (jinhojang6) [reply post.hbs:100]: @jakubgs I thought of this as well. We should have static links for the `apk` and `desktop builds` that link to the latest version. Do you have any solution in mind?

Comment (jakubgs) [reply post.hbs:100]: Yeah, I need to think a bit about that. How to do it properly so we have one place to update this stuff.

Comment (jakubgs) [reply post.hbs:100]: I haven't forgotten about this.

**Comment by jakubgs** [comment]
I'm thinking we could just create sub pages for downloads of releases for individual platforms:
* https://status.im/get/ios/
* https://status.im/get/android/
* https://status.im/get/windows/
* https://status.im/get/macos/
* https://status.im/get/linux/

And it could just contain a single download button with some screenshots, or possible a delayed redirect/download:
```html
<script type="text/javascript">
  setTimeout(function(){ document.location.href = '<%= releaseUrl %>'; }, 5000);
</script>
```

**Comment by jakubgs** [comment]
Or we could just make them immediate redirects:
```html
<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
        <title>Redirecting...</title>
        <link rel="canonical" href="https://status-im-files.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-v1.10.1.apk">
        <meta http-equiv="refresh" content="https://status-im-files.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-v1.10.1.apk">
    </head>
</html>
```

**Comment by jakubgs** [comment]
Okay, I've deployed and merged https://github.com/status-im/status.im/pull/617 we can use the new direct links:

* https://status.im/get/ios/
* https://status.im/get/android/
* https://status.im/get/apk/
* https://status.im/get/linux/
* https://status.im/get/windows/
* https://status.im/get/macos/

**Comment by jakubgs** [comment]
Wait, what?

I completely forgot about this. What's happening? Where am I?

**Pull request closed without merging**